docs/en/reference/sql/dql/WINDOW_CLAUSE.md

```diff
@@ -246,8 +246,8 @@ PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING AND CURRENT ROW
 ```
 
 
-### **Window Exclude Current Time**
-
+### 2. Window Exclude Current Time
```

Comment:
this sentence is incorrect, it should be either one as follows
- Window Excluding Current Time (this explains the meaning)
- WINDOW with EXCLUDE CURRENT_ROW (this describes the syntax)

you may help double check other titles